### PR TITLE
Add values to diagnosis

### PR DESCRIPTION
**Pull request by Aryllen**
Add a bunch of new diagnoses.

This is a draft because I am waiting on a couple.

### CODE REVIEW

File: terms/experimentalData/diagnosis.json [hunk 1]
```diff
@@ -147,6 +147,41 @@
...
             "source": "https://doi.org/10.1186/alzrt254"
+        },
+        {
+            "const": "dementia",
+            "description": "An acquired organic mental disorder with loss of intellectual abilities of sufficient severity to interfere with social or occupational functioning. The dysfunction is multifaceted and involves memory, behavior, personality, judgment, attention, spatial relations, language, abstract thought, and other executive functions. The intellectual decline is usually progressive, and initially spares the level of consciousness.",
+            "source": "http://purl.obolibrary.org/obo/DOID_1307"
+        },
+        {
+            "const": "Parkinson's disease",
+            "description": "A synucleinopathy that has_material_basis_in degeneration of the central nervous system that often impairs motor skills, speech, and other functions.",
```
Comment (cw-dvr8) [inline terms/experimentalData/diagnosis.json:158]: Why does "has_material_basis_in" have underscores?

File: terms/experimentalData/diagnosis.json [hunk 2]
```diff
@@ -147,6 +147,41 @@
...
+            "source": "http://purl.obolibrary.org/obo/DOID_14330"
+        },
+        {
+            "const": "obesity",
+            "description": "An overnutrition that is characterized by excess body fat, traditionally defined as an elevated ratio of weight to height (specifically 30 kilograms per meter squared), has_material_basis_in a multifactorial etiology related to excess nutrition intake, decreased caloric utilization, and genetic susceptibility, and possibly medications and certain disorders of metabolism, endocrine function, and mental illness.",
```
Comment (cw-dvr8) [inline terms/experimentalData/diagnosis.json:163]: Why does "has_material_basis_in" have underscores?

**Review by cw-dvr8 — COMMENTED** [review]
I have a couple of questions about the typography.

**Comment by Aryllen** [comment]
@cw-dvr8, thank you for catching those copy/paste mistakes! I should have looked them over more carefully. I've fixed the weird underscores.

I might have another diagnosis coming soon-ish, but for now this is ready for review.

**Review by avanlinden — APPROVED** [review]

**Pull request merged**